src/core-plugins/CoreCommands.ts

```diff
@@ -24,9 +25,31 @@ export default class CoreCommands extends Plugin {
             }
         });
 
+        this.commands.registerCommand({
+            name: "plugins",
+            description: "List your Aliucord Plugins.",
+            options: [],
+            execute: (args, ctx) => {
+                const enabledPlugins: string[] = [];
+                const disabledPlugins: string[] = [];
+
+                for (const plugin in aliucord.pluginManager.plugins) {
+                    if (aliucord.pluginManager.isEnabled(plugin)) {
+                        enabledPlugins.push(plugin);
+                    } else {
+                        disabledPlugins.push(plugin);
+                    }
+                }
+
+                const plugins = `**Total plugins**: **${Object.keys(aliucord.pluginManager.plugins).length}**\n\n**Enabled plugins**: **${enabledPlugins.length}**\n> ${enabledPlugins.join(", ") ? enabledPlugins.join(", ") : "None."}\n**Disabled plugins**: **${disabledPlugins.length}**\n> ${disabledPlugins.join(", ") ? disabledPlugins.join(", ") : "None."}`;
```

Comment:
I would make this multiline instead of using \n, just to make it more readable
Something like:
```js
const plugins = `
**Total plugins**: ...
**Enabled plugins**: ...
`
```